catalog/app/constants/urls.js

```diff
@@ -8,6 +8,7 @@ export const blogYC =
 export const caseStudy = 'https://quiltdata.com/docs/quants.pdf'
 export const company = 'https://company.quiltdata.com'
 export const docs = 'https://docs.quiltdata.com'
+export const docs35 = 'https://docs.quiltdata.com/v/master'
```

Comment:
why do we need a different url here?

Reply:
There is no https://docs.quiltdata.com/catalog/preferences but https://docs.quiltdata.com/v/master/catalog/preferences

When we deploy a new version (v.3.5) first link will become available https://docs.quiltdata.com/catalog/preferences. And we could find all `docs35` and replace them with `docs` (Catalog Preferences, Athena Queries etc.)

Reply:
maybe call the variable `docsMaster` then?

Reply:
I don't mind to use `docsMaster`.
My reason was that `docsMaster` is a good name and you don't have the itch to replace it with just `docs`.